src/panels/lovelace/editor/config-elements/hui-conditional-card-editor.ts

```diff
@@ -36,7 +42,10 @@ export class HuiConditionalCardEditor extends LitElement
   @property() public hass?: HomeAssistant;
   @property() public lovelace?: LovelaceConfig;
   @property() private _config?: StackCardConfig;
+  @property() private _GUImode = true;
+  @property() private _guiModeAvailable? = true;
```

Comment:
Does this need the optional (undefined) `?` (I don't remember what that is called)

Reply:
The event does not have to send it, so it could be undefined, should not happen...

Reply:
ah Gotcha